packages/bbui/vite.config.mjs

```diff
@@ -1,7 +1,13 @@
-import { defineConfig } from "vite"
 import { svelte } from "@sveltejs/vite-plugin-svelte"
+import { defineConfig } from "vite"
 import path from "path"
 import cssInjectedByJsPlugin from "vite-plugin-css-injected-by-js"
+import { fileURLToPath } from "url"
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+const ignoredWarnings = ["element_invalid_self_closing_tag"]
```

Comment:
NAB: Ah is this the linting rule here? Can you not turn this off in the linting settings instead?